.devcontainer/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Runs once after devcontainer is created
+
+figlet "1-time Setup"
+echo "Running one-time setup for you :)"
+
+# Create .env files if it doesn't exist
+if [ ! -f ./client/.env ]; then
+    cp ./client/.env.example ./client/.env
+fi
+if [ ! -f ./server/.env ]; then
+    cp ./server/.env.example ./server/.env
+fi
+
+# Give python editor intellisense
+(cd server && POETRY_VIRTUALENVS_CREATE=false poetry install)
+
+# Build containers so less waiting
+docker compose pull
```

Comment:
Pull is different from build...

Pull will only pull images works if you don't have dockerfiles, but doesn't not do build.

Probably do both!